src/Results/Results.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './Results.scss'
+
+interface ResultsProps {
+	searchResults: Array<any>
+}
+
+const Results: React.FC<ResultsProps> = props => {
+
+	const noResults = 'I\'m sorry, there are no results. Please try again!'
+
+	const resultsList = props.searchResults.map(result => {
+		return (
+			<article className="results-container" key={result.latitude}>
+				<h2 className="card-header">{result.name}</h2>
+				<h3 className="card-label">{result.label}</h3>
+				<p>Country: {result.country}</p>
+				<p>Continent: {result.continent}</p>
```

Comment:
do we need continent and country anymore since we are focused on America? I think I did USA in my details page and it may be irrelevant now.

Reply:
Yeah, I agree. Let's take them out! We don't get that much very great data back from that first API, so it was hard to pick and choose what to display.